packages/blockchain-link-utils/src/blockbook.ts

```diff
@@ -111,8 +138,10 @@ export const filterEthereumInternalTransfers = (
                     type = 'self';
                 } else if (isIncoming) {
                     type = 'sent';
-                } else {
+                } else if (isOutgoing) {
                     type = 'recv';
+                } else {
+                    type = 'unknown'; // For transactions where both 'from' and 'to' addresses are not the account address (like instant staking)
```

Comment:
I would introduce new type instead of adding it to 'unknown'

Reply:
something like 'external'

Reply:
also thought about it, https://github.com/trezor/trezor-suite/commit/c3586f7e269c64a49eb2481a5b0aedd9af653c5c